### ISSUE TITLE
Add shortcut hints to Translation View

### ISSUE THREAD

**Issue by TfTHacker** [Fevol/obsidian-translate#30]
Feature request: please add to the tranlation view hints for keyboard shortcuts for invoking translation and language swapping. I see already through trial and error that CMD+ENTER invokes the translate button.

Tnx!

![image](https://user-images.githubusercontent.com/69121180/192089187-f53cf9f5-6969-4eac-813f-f3b2cbe32d0d.png)

**Comment by Fevol** [comment]
It's been a while, but I finally got around to this!
I've made a few changes so that triggering the actions via hotkeys is way more obvious:
1. User can set their own hotkeys for every action
2. Shortcut hints will show up in the tooltip
3. These hints can be disabled if the user wishes to do so
I'm considering maybe not showing the hints at all, as the code for it is quite messy

Here's a short GIF detailing some of the future, though I don't know how clear it will be - it's a bit hard to properly show it off:
![shortcuts-example](https://user-images.githubusercontent.com/8179397/202597784-620cc601-031e-476a-8bbc-0b5646905ac2.gif)

The entire update should _hopefully_ be out in the coming days, I'm going to work on one last feature, the glossary; and then start polishing and bugfixing.

I'm really sorry for the long wait!
